.github/workflows/checks.yml

```diff
@@ -142,10 +142,10 @@ jobs:
       - name: post comment
         if: always() && job.status != 'cancelled'
         id: postcomment
-        uses: ./.github/actions/post-comment
+        uses: ./.github/actions/post-comment-v2
         with:
           comment: ${{ steps.comment.outputs.body }}
-          prNumber: ${{github.event.pull_request.number}}
+          sha: ${{ github.sha }}
```

Comment:
possibly should be https://github.com/LedgerHQ/ledger-live-common/blob/master/.github/workflows/bot1.yml#L46